docs/changes/uncertainty/14421.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Ensure that ``Distribution`` can be compared with ``==`` and ``!=``
+with regular arrays or scalars, and that inplace operations like
+``dist[dist<0] *= -1`` work.  This also ensures that the ``Angle`` subclasses
+``Longitude`` and ``Latitude`` can be initialized with a ``Distribution``.
```

Comment:
I'm not sure we want to advertise initializing `Angle` subclasses with `Distribution` quite yet.

Reply:
OK, you're right - let's announce that properly when the statistics make sense!